wgengine/wgengine.go

```diff
@@ -14,6 +14,9 @@ import (
 	"tailscale.com/wgengine/filter"
 )
 
+// ByteCount is the number of bytes that have been sent or received.
+// TODO: why is this a type? for a stringer? where's the stringer?
```

Comment:
This status information from wireguard is half-baked. I don't think this or the next TODO has a good answer. (Or if it does: I don't know what it is.)

Reply:
k, kept but updated TODO comment a bit.